project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_groups.scss

```diff
@@ -461,16 +461,37 @@
 }
 
 .field-name-c4m-ds-content-actions {
-  margin: 15px 0 30px;
+  margin: 15px 0 15px;
 
   .social-actions-share {
     margin-top: 20px;
+    padding: 10px;
+
     h3 {
+      text-transform: uppercase;
       margin-bottom: 5px;
     }
   }
 }
 
+h1 {
```

Comment:
This is `h1.page-header` - there is only one class like this in the page. But I can be a little more specific if you wish.